content/configuration/personal_access_token.md

```diff
@@ -0,0 +1,85 @@
+---
+description: GoCD personal access token
+keywords: GoCD configuration, authentication, access token, token, personal access token, security
+title: Personal access token
+---
+
+# Personal Access Token
+
+From GoCD server version 19.2.0 onwards, you will be able to create personal access tokens to access GoCD API(s). This will allow users to make an API call without specifying their credentials (username & password) as a part of API request headers. 
```

Comment:
This will allow users to make an API call without specifying their credentials (username & password) as a part of API request headers.

Just wondering if we can make it explicit that " Users can now use these tokens to make GoCD API calls and will not be needed to specify their credentials (username & password)".